api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -1235,6 +1235,22 @@ components:
           format: int32
         has_data:
           type: boolean
+    TallyReportTotalMonthly:
+      description: "Tally report monthly summary."
+      required:
+        - date
```

Comment:
The way I read the card, it seems like we would want to remove `date` from `required` but also not have `date` allow nullable.  

As written, the schema validates on entries that look like 
```json
{"value":0.0, "date": null, "has_data":false}
```
or
```json
{"value":0.0, "date": "2022-01-01T00:00:00Z", "has_data":false}
```
but does not validate on JSON like 
```json
{"value":0.0,"has_data":false}
```
(since date is required even if it is allowed to be null).  From the way I read the issue, it's the last example and second to last examples that we need to accept.

Reply:
Ops, sorry. You're totally right. PR updated.